docs/bigquery-usage.rst

```diff
@@ -397,13 +386,17 @@ the job locally:
 .. doctest::
 
    >>> from gcloud import bigquery
+   >>> from gcloud.bigquery import SchemaField
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
-   >>> job = table.load_from_storage(bucket_name='bucket-name',
-   ...                               object_name_glob='object-prefix*',
-   ...                               source_format='CSV',
-   ...                               skip_leading_rows=1,
-   ...                               write_disposition='truncate')
+   >>> table.schema = [
```

Comment:
Filed as an ["internal feature request"][1] just 9 days ago. Seems strange that it would just have come up for the API team.

[1]: https://code.google.com/p/google-bigquery/issues/detail?id=371